src/v0/destinations/google_adwords_offline_conversions/utils.js

```diff
@@ -31,13 +32,49 @@ const {
   trackCallConversionsMapping,
   consentConfigMap,
 } = require('./config');
-const { processAxiosResponse } = require('../../../adapters/utils/networkUtils');
 const Cache = require('../../util/cache');
 const helper = require('./helper');
 const { finaliseConsent } = require('../../util/googleUtils');
+const { HTTP_STATUS_CODES } = require('../../util/constant');
+const { nodeSysErrorToStatus } = require('../../../adapters/utils/networkUtils');
 
 const conversionActionIdCache = new Cache(CONVERSION_ACTION_ID_CACHE_TTL);
 
+const processAxiosResponse = (clientResponse, logger) => {
```

Comment:
Rather than cloning this, we could've written in the original itself. In this way

```js

const processAxiosResponse = (clientResponse, logger) => {
  ...
  logger?.info("I will log this for debugging");
  ...
};
```

Reply:
If I wrote on the original it will log for every other destinations which will include unncessary logs. So I decided to go with this approach.

Reply:
Unless someone passes logger through the function argument we wont log it. If we maintain this logic then logging becomes conditional. I would suggest not to duplicate the code but control logging via flag/ argument

Reply:
I have already added the logger to `nativeIntegration.ts` file, from where this function is being called. I have duplicated this, not to hamper any other implementations.